saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/content/JudgmentContentFileExtractor.java

```diff
@@ -0,0 +1,72 @@
+package pl.edu.icm.saos.importer.notapi.common.content;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import org.apache.commons.io.FilenameUtils;
+import org.springframework.stereotype.Service;
+
+import pl.edu.icm.saos.importer.common.ImportException;
+
+/**
+ * Extractor of judgment content files
+ * 
+ * @author madryk
+ */
+@Service
+public class JudgmentContentFileExtractor {
+    
+
+    /**
+     * Extracts judgment content file from archive file.
+     * 
+     * Looks in the given archiveFile for a file with a name (without extension)
+     * equal to judgmentSourceId.
+     * At the moment only zip archive files are supported.
+     * 
+     * @param archiveFile - archive file from which judgment content file can be extracted
+     * @param judgmentSourceId - source id of judgment.
+     * @return 
+     * @throws IOException 
+     * @throws ImportException
```

Comment:
The exceptions are not described. One is not able to guess when they are thrown without looking in the code.